cfme/common/vm_views.py

```diff
@@ -433,6 +433,19 @@ def is_displayed(self):
         return False
 
 
+class PublishVmView(BaseLoggedInPage):
+    title = Text('#explorer_title_text')
+
+    @View.nested
+    class form(BasicProvisionFormView):
+        submit_button = Button('Submit')
+        cancel_button = Button('Cancel')
+
+    @property
+    def is_displayed(self):
+        return False
```

Comment:
Question: Any check here?

Reply:
I found that without is_displayed returning False the navigation skips some steps and the test fails. I proceeded in a manner you can see a few lines above in MigrateVmView and CloneVmView.
Apart from the title, these views probably can't be checked by is_displayed. They are essentially just filtered options from BasicProvisionFormView. No identification of the view, apart from the title, can be checked to determine that the view is displayed.